symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/MessageUpdateExampleMain.java

```diff
@@ -0,0 +1,35 @@
+package com.symphony.bdk.examples;
+
+import static com.symphony.bdk.core.activity.command.SlashCommand.slash;
+import static com.symphony.bdk.core.config.BdkConfigLoader.loadFromSymphonyDir;
+import static java.util.concurrent.Executors.newSingleThreadExecutor;
+
+import com.symphony.bdk.core.SymphonyBdk;
+import com.symphony.bdk.core.service.message.model.Message;
+import com.symphony.bdk.gen.api.model.V4Message;
+
+import lombok.SneakyThrows;
+
+/**
+ * This demonstrates a basic usage of the message update feature.
+ */
+public class MessageUpdateExampleMain {
+
+  public static void main(String[] args) throws Exception {
+
+    final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
+
+    bdk.activities().register(slash("/go", false, context -> {
+      newSingleThreadExecutor().submit(() -> startTheShow(bdk, context.getStreamId()));
```

Comment:
why do we need the thread executor?

Reply:
To void blocking the DF loop, and run multiple `/go` at the same time. But I just updated the example and removed it